Assets/MRTK/SDK/StandardAssets/Controllers/DebugControllers/Models/webxr-models.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
not: should we rename this folder? I feel like the WebXR part of the name can be misleading, and the license itself clarifies the source.

Reply:
Not sure what we'd rename it to. Would it be PlayCanvas, since that seems to be the ultimate parent project.
https://developer.playcanvas.com/en/tutorials/webxr-controllerhand-models/

The license mentions Amazon, but I can't quite determine where their involvement started/ended.
https://github.com/immersive-web/webxr-input-profiles/blob/main/packages/assets/LICENSE.md

Reply:
Actually looking deeper, it does seem to all originate from WebXR, so I'd say we should stick to that name

Reply:
My comment was less around where they originate from (the license succeeds for that, I think?) and more about relating what they mean to MRTK in the folder structure. They aren't specific to WebXR in the way we use them, so I fear that naming could be confusing.